core/integration_test/hws_integration_test.jl

```diff
@@ -1,4 +1,4 @@
-@testitem "HWS model integration test" timeout = 3600 begin
```

Comment:
Note that I removed this because testitem timeouts are not a feature of TestItemRunner, so we can't add it back. Perhaps we can do something similar in TeamCity itself.